components/vocabularies/api/src/main/java/org/phenotips/vocabulary/VocabularyTerm.java

```diff
@@ -44,6 +44,13 @@
     String getId();
 
     /**
+     * The preferred term identifier for internal storage, for example {@code MIM: 260540} or {@code ENSG00000230986}.
+     *
+     * @return the term identifier, or {@code null} if the term does not have an associated identifier
+     */
+    String getUnderlyingId();
```

Comment:
I am not a fan of this method name. This is the kind of method that should speak for itself, but I wouldn't know what it does without reading the javadoc. Suggestions: `getStoredId`, `getPreferredId`. @sdumitriu thoughts?